Source/qol/chatlog.cpp

```diff
@@ -136,14 +136,14 @@ void AddMessageToChatLog(std::string_view message, Player *player, UiFlags flags
 		for (std::string s; getline(ss, s, '\n');) {
 			lines.push_back(s);
 		}
-		for (int i = lines.size() - 1; i >= 1; i--) {
+		for (size_t i = lines.size() - 1; i >= 1; i--) {

```

Comment:
This doesn't do the right thing if `lines.size()` is 1 or 0.

Reply:
Could you elaborate?

From my understanding we always have some sort of text (at leas the timestamp and player info is added), that means lines should always be >= 1.
If lines is 1, the for-loop is not entered (as before), because the first line is always added explicitly (line 143).

Reply:
still, it would be best not to add traps like these to the code :D

Reply:
It iterates all lines expect the first one in reverse order. So it should work. 🤷‍♂️ 

But anyway if preferred, we can stick with a int and add a cast to `lines.size()`. Should I do so?

Reply:
I changed it to a int cast. 😉